base/allocator/partition_allocator/shim/allocator_shim.cc

```diff
@@ -405,6 +405,10 @@ ALWAYS_INLINE void ShimAlignedFree(void* address, void* context) {
 #else  // BUILDFLAG(USE_PARTITION_ALLOC_AS_MALLOC)
 #include "base/allocator/partition_allocator/shim/allocator_shim_override_mac_symbols.h"
 #endif  // BUILDFLAG(USE_PARTITION_ALLOC_AS_MALLOC)
+#elif (BUILDFLAG(IS_STARBOARD) && defined(_LIBCPP_HAS_MUSL_LIBC))
+// // In file included from ../../base/allocator/partition_allocator/shim/allocator_shim.cc:409:
```

Comment:
Do we need the copy of the error messages here ? It's quite a lot of visual noise. If you need this for posterity, it's probably better to keep this in the bug and have this just clean with the last comment line that you have